project/plugins.sbt

```diff
@@ -1,5 +1,11 @@
 scalacOptions += "-deprecation"
 
+addSbtPlugin("org.scala-native" % "sbt-crossproject" % "0.1.0")
```

Comment:
sbt-scala-native depends on sbt-crossproject you can remove this line if you want to.